### PR TITLE
HDDS-2796. beyond/_index.md translation

### PR DESCRIPTION
**Pull request by iamabug**
## What changes were proposed in this pull request?

doc added

## What is the link to the Apache JIRA

https://issues.apache.org/jira/browse/HDDS-2796

## How was this patch tested?

hugo server -D

### CODE REVIEW

File: hadoop-hdds/docs/content/beyond/_index.zh.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+-->
+
+{{<jumbotron title="进阶">}}
+  本部分介绍 Ozone 的自定义配置，包括如何将 Ozone 以并存的方式部署到已有的 HDFS 集群，以及如何运行 Ozone 内置的 profilers 和追踪报告。
```
Comment (iamabug) [inline hadoop-hdds/docs/content/beyond/_index.zh.md]: I am not sure about this.

Comment (cxorm) [inline hadoop-hdds/docs/content/beyond/_index.zh.md]:
```suggestion
  本部分介绍 Ozone 的自定义配置，包括如何将 Ozone 以并存的方式部署到已有的 HDFS 集群。以及如何运行 Ozone 内置的 profilers 和 tracing 功能。
```

**Comment by adoroszlai** [comment]
@xiaoyuyao could you please review?

**Comment by xiaoyuyao** [comment]
LGTM, 1+.

**Pull request merged**